src/test/java/io/jenkins/plugins/grading/CoverageScoreTests.java

```diff
@@ -14,15 +12,45 @@
  *
  * @author Eva-Maria Zeintl
  * @author Ullrich Hafner
+ * @author Andreas Riepl
  */
 class CoverageScoreTests {
+
     @Test
-    void shouldCalculate() {
+    void givenCoverageScore_whenRatio99Percent_thenTotalImpactShouldEqualImpactValue() {
```

Comment:
keine `_` verwenden

Reply:
Gibt es einen bestimmten Grund hierfür? Ich dachte, dass dieser given_when_then Style State-Of-The-Art wär

Reply:
In Java ist die Konvention, keine Underscores zu nutzen, das ist der einzige Grund. Wenn Sie einen sprechenden Namen vergeben wollen, geht das mit `@DisplayName`. 

Für die Benennung von Methoden bin ich offen.